server/main.py

```diff
@@ -51,12 +51,24 @@ async def receive(self) -> dict:
         return await self._websocket.receive_json()
 
     def __eq__(self, other: object) -> bool:
+        """Compares the Client to another object.
+
+        If the object is not an instance of Client, NotImplemented is returned.
+
+        Args:
+            other: The object to compare the Client to.
+
+        Returns:
+            True of the ID of the client is equal to the other client's ID,
```

Comment:
`True if`

Reply:
Can probably use `id` as well since that is how we have cased the instance attribute